docs/administrative/change_owner.rst

```diff
@@ -1,46 +1,59 @@
 .. _change_owner:
 
 ===================================
-Change your cloud account owner
+Change your Cloud Account Owner
 ===================================
 
-You might need to change the primary user, also known as the account owner,
-when a user leaves the company or the company or
-project name changes. Update this information for
+When a Primary User/Account Owner leaves the company or the name of the company or
+project changes, then you need to change the Primary User/Account Owner. 
+To Update this information for
 your Rackspace Technology Cloud and Amazon Web Services® (AWS)
-accounts through
-the `Rackspace Technology Customer Portal <https://login.rackspace.com>`_.
+accounts, Login to `Rackspace Technology Customer Portal <https://login.rackspace.com>`_.
 
-.. note::
+**Note: Only the current Acount Owner can assign a New Account Owner.**
 
-      Only the current account owner can assign a new account owner.
-
-When you change the account owner:
+**When you Change the Account Owner**:
 
 - You must select the new or promoted user
-  from existing users on the account. The selected
-  user becomes the primary contact and has full control over the account.
+  from existing users on the account. 
+
+- The selected user becomes the Primary Contact and has full control over the account.
 
 - As the former or demoted primary user, you are set
   up as an Administrative contact type and given
   basic `Role-Based Access Control (RBAC)
   Identity permissions <https://developer.rackspace.com/docs/cloud-identity/v2/getting-started/>`_.
   The primary user might need to set up
-  RBAC permissions for the demoted user, depending on the use case.
+  RBAC permissions for the demoted user.
+
+
+You can **Change the Account Owner** by using the following steps:
+
+**Step 1** Log in to the `Rackspace Technology Customer Portal <https://login.rackspace.com>`_.
+
+**Step 2** In the global navigation menu, click **Account** >> **Account Settings**.
+
+.. image:: accountsetting.png
+   :width: 590
+   :alt: accountsetting 
```

Comment:
alt tag should be regular text that people can read when hovering over the image or for screen reader to describe. Like make this a short fragment incomplete sentence with a few words that describe what is happening in the image.